torch/_lazy/__init__.py

```diff
@@ -34,3 +35,15 @@ def sync_multi(tensors, devices):
 def get_tensor_id(tensor):
     """Return a unique id of the lazy tensor maintained by LTC"""
     return torch._C._lazy._get_tensor_id(tensor)
+
+
+def to_cpu(tensors, devices=None):
+    devices = devices or ["lazy"]
+
+    flattened, spec = tree_flatten(tensors)
```

Comment:
`tree_map` would be a good fit for your purpose here. Example code, https://github.com/pytorch/pytorch/blob/abcf01196cd27805349aa892db847f9a61f52c0e/functorch/functorch/_src/aot_autograd.py#L212

Reply:
I specifically chose to manually flatten and unflatten as I need the flattened version to call `sync_multi` and it seemed unnecessary to call "flatten" again indirectly via `tree_map`